data/json/recipes/weapon/mods.json

```diff
@@ -104,6 +104,65 @@
     "tools": [ [ [ "large_repairkit", 50 ], [ "small_repairkit", 100 ] ] ],
     "components": [ [ [ "u_shotgun", 1 ] ], [ [ "scrap", 3 ] ], [ [ "plastic_chunk", 1 ] ] ]
   },
+  {
+    "result": "makeshift_pistol_bayonet",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": 500,
+    "reversible": true,
+    "autolearn": true,
+    "using": [ [ "cordage_short", 2 ] ],
+    "components": [ [ [ "spike", 1 ] ] ]
+  },
+  {
+    "result": "makeshift_sword_bayonet",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": 500,
+    "reversible": true,
+    "autolearn": true,
+    "using": [ [ "cordage", 1 ] ],
+    "components": [ [ [ "makeshift_machete", 1 ], [ "blade", 1 ], [ "machete", 1 ], [ "survivor_machete", 1 ] ] ]
+  },
+  {
+    "result": "pistol_bayonet",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": 320000,
+    "book_learn": [ [ "recipe_melee", 4 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 3 }, { "id": "CHISEL", "level": 3 } ],
+    "tools": [
+      [ [ "tongs", -1 ] ],
+      [ [ "anvil", -1 ] ],
+      [ [ "swage", -1 ] ],
+      [ [ "forge", 250 ], [ "oxy_torch", 50 ] ],
+      [ [ "mold_plastic", -1 ] ]
+    ],
+    "components": [ [ [ "steel_chunk", 2 ], [ "scrap", 6 ] ], [ [ "plastic_chunk", 1 ] ] ]
+  },
+  {
+    "result": "sword_bayonet",
+    "type": "recipe",
+    "category": "CC_WEAPON",
```

Comment:
Recipe meant as override of vanilla, would cause duplicate recipe entry w/o ``id_suffix`` and it's probably better to unify this and the regular one into a single recipe(or change the vanilla one to this.)